src/app/Shared/Components/FileUploads.tsx

```diff
@@ -258,7 +257,11 @@ export const MultiFileUpload: React.FC<MultiFileUploadProps> = ({
 
   return (
     <>
-      <Prompt when={uploading} message="Are you sure you wish to cancel the file upload?" />
+      {/* 
+        TODO: Add back when supported 
```

Comment:
Can we work around this in the meantime?

https://stackoverflow.com/questions/70178993/alternative-for-prompt-in-react-router-v6

Reply:
Hmm unfortunately, that requires the `Prompt` to be under a `Router`. 

```
useBlocker must be used within a data router.  See https://reactrouter.com/routers/picking-a-router.
    at invariant (webpack-internal:///12599:280:11)
    at useDataRouterContext (webpack-internal:///89250:672:85)
    at useBlocker (webpack-internal:///89250:814:7)
    at Prompt (webpack-internal:///5438:29:74)
    at renderWithHooks (webpack-internal:///73116:14985:18)
    at mountIndeterminateComponent (webpack-internal:///73116:17811:13)
    at beginWork (webpack-internal:///73116:19049:16)
    at HTMLUnknownElement.callCallback (webpack-internal:///73116:3945:14)
    at Object.invokeGuardedCallbackDev (webpack-internal:///73116:3994:16)
    at invokeGuardedCallback (webpack-internal:///73116:4056:31)
```

Right now, this `FileUploads` are used on modals, which are mounted on a node `rootPortal` outside root node.

https://github.com/cryostatio/cryostat-web/blob/316e9c7cb8c27b125a23a80b95e3e0fd25faa9c1/src/index.html#L30

I believe it was done that way to ensure modals are always on top.

Reply:
We can exploring movign `portal-root` down to below `Router` but I am not sure how that gonna break modals. Would u prefer to have it as a separate issue for more research?

Reply:
Ahh it needs the `createBrowserRouter` and `RouterProvider` way to be able to use.

https://reactrouter.com/en/main/routers/picking-a-router#using-v64-data-apis

Reply:
Okay, let's create a separate issue to track it. I don't think this is that big of a loss to spend a ton of time on or to block the version upgrade until this is fixed.

Reply:
#1147